ui/.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Edge (localhost)",
+            "type": "msedge",
```

Comment:
```suggestion
            "type": "pwa-msedge",
```

Reply:
(I have no idea what difference it makes, it was suggested to me recently as the new way to launch edge for React apps.